cmd/helm/release_testing.go

```diff
@@ -66,6 +67,7 @@ func newReleaseTestCmd(c helm.Interface, out io.Writer) *cobra.Command {
 	f := cmd.Flags()
 	f.Int64Var(&rlsTest.timeout, "timeout", 300, "time in seconds to wait for any individual Kubernetes operation (like Jobs for hooks)")
 	f.BoolVar(&rlsTest.cleanup, "cleanup", false, "delete test pods upon completion")
+	f.BoolVar(&rlsTest.parallel, "parallel", false, "run test pods in parallel")
```

Comment:
Curious about if this should be true by default? It seems like there are no downsides to running the tests in parallel, right?

Reply:
I think in general defaults should be to stick to existing behaviour, you never know what setups people have.

You could for example have 2 tests that do similar setup/teardown on the database, that when run in parallel conflict with each other